### PR TITLE
Update: "Staking State" in Transaction Accounting Model

### PR DESCRIPTION
**Pull request by lezzokafka**
Adding examples and slashing related information.

### CODE REVIEW

File: docs/getting-started/transaction-accounting-model.md [hunk 1]
```diff
@@ -27,16 +27,44 @@ Different types of transactions and how they relate to these accounting are [des
...
 
 - bonded amount
 - unbonded amount
 
-The bonded amount is the amount used to check against minimal staking requirements and used to calculate the Tendermint validator voting power (in case of council nodes).
-
-As it may take time for the network evidence of malicious activity (e.g. double signing) to appear, the stake cannot be withdrawn immediately and is first moved to the “unbonded” balance.
-
-The unbonded balance can be withdrawn (into transaction outputs) after “unbonded_from” time if the account was not jailed / slashed (see [staking](./staking)).
+and and its slashing related information.
```
Comment (tomtau) [inline docs/getting-started/transaction-accounting-model.md]:
```suggestion
and its slashing related information. 
```

File: docs/getting-started/transaction-accounting-model.md [hunk 2]
```diff
@@ -27,16 +27,44 @@ Different types of transactions and how they relate to these accounting are [des
...
+| Unbonded From   | 2020-02-02 08:28:16 +08:00 |
++-----------------+----------------------------+
+| Jailed Until    |                 Not jailed |
++-----------------+----------------------------+
+| Punishment Type |               Not punished |
++-----------------+----------------------------+
+| Slash Amount    |               Not punished |
++-----------------+----------------------------+
+```
+- The `Nonce` is the number of transactions involved with the account.
```
Comment (tomtau) [inline docs/getting-started/transaction-accounting-model.md]: not only transactions, but also events.

**Review by tomtau — CHANGES_REQUESTED** [review]
it's ok, just a typo + one comment

File: docs/getting-started/transaction-accounting-model.md (hunk 2 above)
Comment (lezzokafka) [reply docs/getting-started/transaction-accounting-model.md]: Oh right! Thanks 👍

**Pull request merged**